presto/README.md

```diff
@@ -0,0 +1,28 @@
+# Presto
+
+Presto is a distributed SQL query engine for big data.
+- [Github](https://github.com/prestodb/presto)
+- [Homepage](https://prestodb.io)
+
+The benchmarks are based on Presto version `0.287`.
+
+We assume that a Presto cluster is already running. For more information, visit [Getting Started](https://prestodb.io/getting-started/).
```

Comment:
Need to add instructions on how to install Presto on an empty machine with Ubuntu on AWS.

Reply:
It's probably not possible to have a single script to set up a cluster since the environment is complicated, what should I do?

Reply:
Having a standard setup is important otherwise different setups may not give identical results.

Would be nice if you can do either of the following:
- You can create an AWS AMI and add that AMI id in the README. 
- You can setup a docker-compose based setup of Presto.

Reply:
@allenshen13 As the other commenters said already, `benchmark.sh` is supposed to be as automatic as possible. This ensures reproducibility and comparability with other systems.

Looking at https://prestodb.io/docs/current/installation/deploy-docker.html, it seem fairly easy to deploy a single-node Presto cluster. The JVM config file could be stored within /presto and copied to the right place by "benchmarks.h". It would be great to have Presto included, would you like to continue this PR?